main.cpp

```diff
@@ -65,10 +63,26 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int count = 1;
+    int type = 5;
+
+    char whatIsMyName[] = "james";
+    char synthesizerType[] = "Roland";
+    char houseName[] = "The Mooring";
```

Comment:
couple comments for this:
1) normally I want a single char, but you provided arrays somewhat correctly, which is cool.
2) these should be `const char`, not `char`.  you'll learn about `const` later, but basically if it's not const and you write more characters to the array than are allowed, you'll have a problem on your hands.  
3) this is C++, not C.  we don't use raw arrays.  you'll learn the replacement when templates are covered.